fixtures/User.yml

```diff
@@ -3,4 +3,4 @@ items:
   user{1..100}:
     firstName: '{{name.firstName}}'
     lastName: '{{name.lastName}}'
-    age: 22
+    age: "{{random.number}}"
```

Comment:
Why ?

Reply:
It is better to put a random here than just a fixed number

Reply:
But it doesn't make sense to put a random number, which will most of the time be > 100, so trash data. Maybe rebase on the authentication branch, and make it null